tutorial/cytoscape/layout_chapter.py

```diff
@@ -170,15 +171,15 @@
     '''),
 
     dcc.Markdown(dedent('''
-    
+
     ## Fine-tuning the Layouts
-    
-    For any given `name` item, a collection of keys are accepted by the layout 
-    dictionary. For example, the 
-    [`grid` layout](http://js.cytoscape.org/#layouts/grid) 
-    will accept `row` and `cols`, the 
-    [`circle` layout](http://js.cytoscape.org/#layouts/circle) accepts `radius` 
-    and `startAngle`, and so forth. Here's is the grid layout 
+
+    For any given `name` item, a collection of keys are accepted by the layout
+    dictionary. For example, the
+    [`grid` layout](http://js.cytoscape.org/#layouts/grid)
+    will accept `row` and `cols`, the
+    [`circle` layout](http://js.cytoscape.org/#layouts/circle) accepts `radius`
```

Comment:
```suggestion
    [`circle` layout](http://js.cytoscape.org/#layouts/circle) `radius`
```
(or "will accept" as in the previous proposition)